presto-plugin-toolkit/src/main/java/io/prestosql/plugin/base/security/FileBasedAccessControl.java

```diff
@@ -343,4 +367,15 @@ private boolean isSchemaOwner(ConnectorSecurityContext context, String schemaNam
         }
         return false;
     }
+
+    private boolean checkSchemaAnyPermission(ConnectorSecurityContext context, String schemaName)
+    {
+        for (SchemaAccessControlRule rule : schemaRules) {
+            Optional<Boolean> owner = rule.match(context.getIdentity().getUser(), schemaName);
+            if (owner.isPresent()) {
```

Comment:
when user is not an owner of a table it does mean they have access to it

Reply:
yes, so only check isPresent, instead of actual value true or false, correct ?

Reply:
I don't understand why. When user is not an owner then which privilege gives him an access to given schema?